dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/preheat/mappers/OrganisationUnitMapper.java

```diff
@@ -28,15 +28,13 @@
 package org.hisp.dhis.tracker.preheat.mappers;
 
 import org.hisp.dhis.organisationunit.OrganisationUnit;
-import org.mapstruct.BeanMapping;
-import org.mapstruct.Mapper;
-import org.mapstruct.Mapping;
-import org.mapstruct.Named;
+import org.mapstruct.*;
```

Comment:
We don't use `*` imports. Can you please configure Intellij not to make such changes 😄